packages/react/src/Button/Button.client.tsx

```diff
@@ -0,0 +1,47 @@
+import "@axa-fr/design-system-css/dist/Button/Button.client.scss";
+import classNames from "classnames";
+import { ComponentPropsWithoutRef, PropsWithChildren, ReactNode } from "react";
+
+export enum Variants {
+  primary = "primary",
+  business = "primary-business",
+  secondary = "secondary",
+  secondaryError = "secondary-error",
+  tertiary = "tertiary",
+  ghost = "ghost",
+}
+
+type ButtonProps = {
+  variant?: Variants;
+  iconLeft?: ReactNode;
+  iconRight?: ReactNode;
+} & ComponentPropsWithoutRef<"button">;
+
+export const ButtonClient = ({
+  children,
+  className,
+  disabled,
+  variant,
+  iconLeft,
+  iconRight,
+  ...args
+}: PropsWithChildren<ButtonProps>) => {
+  return (
+    <button
+      className={classNames(
+        "af-btn-client",
+        variant && `af-btn-client--${variant.toString()}`,
+        disabled && "af-btn-client--disabled",
+        iconLeft ? "af-btn-client--hasiconLeft" : undefined,
+        iconRight ? "af-btn-client--hasiconRight" : undefined,
```

Comment:
```suggestion
        iconLeft && "af-btn-client--hasiconLeft",
        iconRight && "af-btn-client--hasiconRight" ,
```
classNames n'ajoutera pas les classes si iconLeft/Right sont undifined